demo-invenioils/ui/src/overridden/CustomBanner.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Message } from 'semantic-ui-react';
+
+export const CustomBanner = () => {
```

Comment:
not sure i understand the need to implement this, don't we have always a banner in the fixtures?